deps/oob-1.0.2/jsonutil.c

```diff
@@ -217,6 +219,8 @@ char *prepareOOBevent(oobOcspData *ocspevent)
   vals = cJSON_CreateObject();
   cJSON_AddItemToObject(list, "Value", vals);
 
+  cJSON_AddItemToObject(vals, "DSN", dsn);
```

Comment:
Why not check ths is null or not?

Reply:
After the usage of dsn, we should free the object.

Reply:
@sfc-gh-azhan This dsn object becomes part of cJSON* root which is freed after preparing the oobevent and converted to string. 
Line 320 in jsonutil.c 

```
  if (root)
  {
    str = cJSON_Print(root);
    cJSON_Delete(root);
  }
  freeAll();
  return str;
```

Reply:
As discussed, the test_oob test case within test_unit_oob.cpp covers the case when dsn is null; it simply does not add this object to the overall oob payload